### DISCUSSION TITLE
Reverse DNS does not match SMTP Banner

### DISCUSSION THREAD

**Discussion by thepra** [emersion/maddy#571] [Q&A]
From mxtoolbox.com at the SMTP Banner Check I get a warning which says: Reverse DNS does not match SMTP Banner
But looking around the web I didn't get what's it about and how should I fix it, does anyone have any idea of what's going on?

**Comment by foxcpp** [comment]
A SMTP server sends its hostname as part of the banner message. For maddy it is defined via `hostname` directive `($(hostname) = example.org`).   

Your servers IP address should have a PTR record containing that hostname (look for Reverse DNS in your providers control panel). It should also resolve correctly back to the same IP address.
